src/Feed.php

```diff
@@ -25,7 +26,7 @@ public function __construct($title, $url, $resolver)
         $this->items = $this->resolveItems($resolver);
     }
 
-    public function toResponse($request)
+    public function toResponse($request): Response
```

Comment:
`toArray(): array` 😈

Reply:
Yeah, little bit silly. But I really want return type hints on the other functions in the file and be consistent about it in the same file.